docs/community/contribute.rst

```diff
@@ -153,12 +153,28 @@ To understand how the below components interact with each other, refer to :ref:`
     * - **Purpose**: Deployment, Documentation, and Issues
     * - **Languages**: RST
 
-To build the Flyte docs locally you will need the following prerequisites:
+In the ``flyteorg/flyte`` root directory you can run ``make dev-docs`` to build the documentation locally. The generated documentation will be in the ``docs/_build/html`` directory.
 
-* Install `conda-lock <https://github.com/conda/conda-lock>`__.
```

Comment:
Can we leave these steps in as an alternative to the steps below? You can move them down and put them in a "Alternative conda-lock setup steps" section.

Reply:
I can add the following steps back as an alternative.

https://github.com/flyteorg/flyte/blob/84c0fefef78ad603f26f40ea0ae2cac4d2c32711/docs/community/contribute.rst?plain=1#L156-L164

Reply:
Done.

Reply:
Thanks!